generative_data_prep/utils/logger.py

```diff
@@ -0,0 +1,94 @@
+"""Copyright 2023 SambaNova Systems, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+
+This class creates a common logger.
+"""
+import datetime
+import logging
+import logging.config
+import os
+
+import git
+
+logging.config.fileConfig("generative_data_prep/utils/logger_config.ini")
```

Comment:
logging configuration should only exist in one place i.e. in `main`; after that it is simply; this config shouldn't be here.